docs/AI-for-security/knowledge-base.asciidoc

```diff
@@ -0,0 +1,139 @@
+
+Refer to the following video for an example of adding a document to knowledge base from the settings menu.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/rQsTujEfikpx3vv1vrbfde.jpg"
+  data-uuid="rQsTujEfikpx3vv1vrbfde"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+[[knowledge-base-add-knowledge-index]]
+=== Add an index
+
+Add an index as a knowledge source when you want new information added to that index to automatically inform AI Assistant's responses. Common security examples include asset inventories, network configuration information, on-call matrices, threat intelligence reports, and vulnerability scans. 
+
+IMPORTANT: Indexes added to knowledge base must have at least one field mapped as {ref}/semantic-text.html[semantic text].
+
+. To open Security AI settings, use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field] to find "AI assistant for security". Go to the Knowledge Base tab.
+. Click **New → Index**.
+. Name the knowledge source.
+. Under **Sharing**, select whether this knowledge should be **Global** or **Private**.
+. Under **Index** enter the name of the index you want to use as a knowledge source.
+. Under **Field**, enter the names of one or more semantic text fields within the index.
+. Under **Data Description**, describe when this information should be used by AI Assistant.
+. Under **Query Instruction**, describe how AI Assistant should query this index to retrieve relevant information.
+. Under **Output Fields**, list the fields which should be sent to AI Assistant. If none are listed, all fields will be sent.
+
+image::images/knowledge-base-add-index-config.png[Knowledge base's Edit index entry menu]
+
+Refer to the following video for an example of adding an index to knowledge base.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/Q5CjXMN4R2GYLGLUy5P177.jpg"
+  data-uuid="Q5CjXMN4R2GYLGLUy5P177"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+[[knowledge-base-rbac]]
+== Role-based access control (RBAC) for knowledge base
```

Comment:
This seems pretty important and should probably be higher up in the doc rather than at the end.